spec/u-namespace.md

```diff
@@ -12,11 +12,14 @@ manages the specification for this namespace, hence the _namespace_ `u:`.
 
 ## Options
 
-This section describes common **_<dfn>`u:` options</dfn>_** which each implementation SHOULD support
+This section describes common **_<dfn>`u:` options</dfn>_** that apply
 for all _functions_ and _markup_.
 
 ### `u:id`
 
+Implementations providing a formatting target other than a concatenated string
```

Comment:
What does it imply for implementations to say that u:id is supported? We should say _something_ about the intended usage.

Reply:
We should insert the examples from the design doc